### ISSUE TITLE
Unable to connect to SQL Server named instance - uses Windows authentication without permission?

### ISSUE THREAD

**Issue by OmegaPhil** [mkleehammer/pyodbc#163]
I'm currently looking into using pyodbc 3.1.1 in a Windows Python 3.5.2 environment to connect to remote SQL Servers (mainly 2012) using standard SQL server credentials (not Windows authentication).

Connecting to a local instance works:

```
pyodbc.connect('DRIVER={SQL Server};SERVER=<local machine>;DATABASE=<database name>;UID=<user>;PWD=<password>')
```

However when I try then to connect to a remote server named instance (e.g. '10.0.0.1\NAME'), the connect call appears to time out and fails with:

```
pyodbc.Error: ('08001', '[08001] [Microsoft][ODBC SQL Server Driver][DBNETLIB]SQL Server does not exist or access denied. (17) (SQLDriverConnect)')
```

With wireshark, I can see an initial UDP packet going out (presumably to the SQL Server Browser to list named instances), but then lots of SMB/SMB2 packets apparently associated with Windows authentication, lots of STATUS_LOGON_FAILUREs etc.

I'm not using 'Integrated SSPI', 'Trusted_Connection' etc, so no Windows authentication should be tried - any idea what is going on?

Thanks

**Comment by mkleehammer** [comment]
I don't.  It's not inherent to pyodbc so I'm marking it as Not A Bug for now.

You might consider changing your driver to a more specific version {SQL Server Native Client 11.0} or something, but I don't know if that really changes the driver or just behavior.

**Comment by OmegaPhil** [comment]
OK, so pyodbc is just considered a thin wrapper around the 'drivers' then - I have tried with the named driver, this time at least it didn't appear to use Windows authentication, just timed out sending UDP packets. Looking around SQL Management Studio's connection, I noticed that in Server Properties the instance was named 'SQLBESPOKE\SQLBESPOKE', yet the details used to login just had 'BESPOKE' for the name.

Using 'SQLBESPOKE' as the instance name suddenly worked with both drivers - so this problem was a combination of some sort of instance name mangling happening in the background, and the driver invalidly silently falling back to Windows authentication.

Thanks for your help.
